docs/content/manual/release-specific/v1.1.1/delete-node.md

```diff
@@ -25,15 +42,17 @@ kubectl create -f https://raw.githubusercontent.com/longhorn/longhorn/v1.1.0/exa
 
 **When** node-1 deleted from the browser.
 
-**Then** should see node-1 removed from the node list in the browser.
+**Then** click `OK` in the pop-up window for delete confirmation.
+
+*And* should see node-1 removed from the node list in the browser.
 
 *And* should see node-1 removed from nodes.longhorn.io.
 ```
 kubectl -n longhorn-system get nodes.longhorn.io
 ```
 
 
-## Test Node Delete - should fail when volume replica not evicted on node
+## Test Node Delete - should delete the node when the node is down, even the schedule is enabled and replicas not evicted from the node
```

Comment:
If the a replica exists on the node and the volume is detached. In that case Longhorn fails to remove the replica and finally fails to remove the node. 
Should we make a note about it?

Reply:
From the test, the volume became degraded and longhorn will remove the node and the replica. I've updated the commit by having more validation steps and examples.

Reply:
Like we discussed, we probably still need to make note for the 1 replica detached volume case.

Reply:
Thanks @innobead , I've missed the case for 1 replica.

Reply:
Added.